gaphor/codegen/tests/test_coder.py

```diff
@@ -261,16 +269,24 @@ def test_coder_write_association_opposite_not_navigable(
     assert a == ['A.b = association("b", B)']
 
 
-def test_attribute_from_super_model(uml_metamodel: ElementFactory):
+@pytest.mark.xfail(reason="Needs further investigation")
+def test_attribute_from_super_model(
+    uml_metamodel: ElementFactory, core_metamodel: ElementFactory
+):
```

Comment:
TODO: find out why this test fails now

Reply:
order matters: core model should be declared first, since the code generator is looking for the _first_ match (by name) in a super-model).